lib/smart_answer_flows/am-i-getting-minimum-wage/questions/past_payment_date.govspeak.erb

```diff
@@ -3,6 +3,7 @@
 <% end %>
 
 <% options(
+  "2017-04-01": "Apr 2017 - Sep 2017",
```

Comment:
@ikennaokpala This option needs to be for "Oct 2016 - Mar 2017". 
"Apr 2017 - Sep 2017" will the `current period` that's checked in the previous question.

We started applying 6 months date ranges to the past payments options, because the National Living Wage.
When it was introduced, the National Living Wage was only updated in April, but the National Minimum Wage was updated in October. To get around this we added an option to allow users to check the rates between October and March.